package.json

```diff
@@ -21,7 +21,16 @@
     "reselect": "^3.0.1"
   },
   "devDependencies": {
+    "@hellofresh/eslint-config-senf": "~4.0.2",
+    "eslint-config-senf": "https://github.com/hellofresh/eslint-config-senf.git",
```

Comment:
Just took it from here... was fighting some time to make it works

![image](https://user-images.githubusercontent.com/8372070/28666303-1b06c992-72c7-11e7-8c65-77fbe52d6010.png)